pkg/k8s/ipresolver.go

```diff
@@ -161,23 +193,68 @@ func (resolver *K8sIPResolver) StartWatching() error {
 				deploymentsWatcher.Stop()
 				cronJobsWatcher.Stop()
 				return
-			case podEvent := <-podsWatcher.ResultChan():
+			case podEvent, ok := <-podsWatcher.ResultChan():
+				podsCounter.Inc()
```

Comment:
Should be after the !ok check

Reply:
it's a counter for the number of events, not resets.  I added now a different counter for that

Reply:
It still needs to be after :) You only want to count events where "ok" was returned, otherwise you'll count non successful events too

Reply:
you're right, fixed